health_check/views.py

```
@@ -29,8 +29,9 @@ def _run(plugin):
                 connection.close()
 
         with ThreadPoolExecutor(max_workers=len(plugins) or 1) as executor:
-            for ers in executor.map(_run, plugins):
-                errors.extend(ers)
+            for plugin, ers in zip(plugins, executor.map(_run, plugins)):
+                if plugin.critical:
```

This is not what I was going for. Now you just ignore errors if they are not critical. So they will be not shown.

---

Correct, as the `errors` list here is only used to determine if we're going to return 500 or 200, and we discussed above that non-critical plugins would would output there errors (see pretty_status method) but not cause the entire endpoint to 500.